integrations/event-handler/fluentd_client.go

```diff
@@ -67,11 +68,13 @@ func NewFluentdClient(c *FluentdConfig, log *slog.Logger) (*FluentdClient, error
 				RootCAs:      ca,
 				Certificates: certs,
 			},
+			MaxIdleConnsPerHost: c.FluentdMaxConnections,
```

Comment:
Should we also enable http2 if the upstream service supports it?

Reply:
The way I read https://pkg.go.dev/net/http#Transport, http/2 should be enabled automatically since we aren't specifying a custom value for `TLSNextProto`, no?

Reply:
I don't think so. Default transport supports http2 but for other transports you must manually enable it using http2 package

>  The [DefaultTransport](https://pkg.go.dev/net/http#DefaultTransport) supports HTTP/2. To explicitly enable HTTP/2 on a transport, use golang.org/x/net/http2 and call ConfigureTransport. See the package docs for more about HTTP/2.

Reply:
TIL.  Fixed.

Reply:
thanks